website/docs/docs/build/incremental-models.md

```diff
@@ -255,6 +255,24 @@ to build incremental models.
 
 Click the name of the adapter in the below table for more information about supported incremental strategies.
 
+The `merge` strategy is available in dbt-postgres and dbt-redshift beginning in dbt v1.6.
+
+<VersionBlock lastVersion="1.5">
+    
```

Comment:
added an extra line @dbeatty10 because the preview was looking off

![Screenshot 2023-06-30 at 14 25 53](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/b14adbf4-4a17-401d-8373-c3d12f06fa88)

Reply:
@mirnawong1 Thanks for always improving my docs PRs by updating the wording or fixing where something is broken. You rock 🎸